lighthouse-core/audits/is-on-https.js

```diff
@@ -72,6 +72,11 @@ class HTTPS extends Audit {
       extendedInfo: {
         formatter: Formatter.SUPPORTED_FORMATS.URL_LIST,
         value: insecureRecords
+      },
+      details: {
```

Comment:
> how about datagrid where the first column is a URL? URL will likely need its own render as it'll get forked for devtools revealing.

or when URL is only sometimes a URL like in stylesheets that are inline and have previews. I think this is a very clean way to handle these situations and is a great step forward from the hackiness of old formatters. 

Also +1 to keeping `extendedInfo` and `details` separate for Paul's reasons and an aspiration to keep the presentation and raw data as separate as possible. All for renaming to make the intention of each object clearer though :)